inst/extdata/OSD/L/LAVELDO.json

```diff
@@ -132,8 +132,8 @@
         "dry_hue": "NA",
         "dry_value": "NA",
         "dry_chroma": "NA",
-        "moist_hue": "7.5YR",
-        "moist_value": 5,
+        "moist_hue": "1OYR",
```

Comment:
This is an example of a series where first color is dry color, but there is no statement identifying that anywhere in the OSD, so it defaults to moist and misses the actual moist color.

Reply:
I should say, the moisture statement only exists inline, which we do not try to parse ,and there is no "Colors are for dry soil ...." generic statement for the typical pedon